ObjectMotion puck_motion
ObjectMotion mallet_motion
Trajectory[] puck_trajectory
Trajectory[] mallet_trajectory
std_msgs/Header header
